Unix/doc/setup-ntlm-omi.md

```diff
@@ -0,0 +1,163 @@
+#Enabling SPNEGO/NTLM Authentication on OMI
+
+OMI requests between the client and server have been up to this point
+authenticated via Basic auth. Basic is very simple, but not at all secure, 
+inasmuch as the username and password are transmitted in the clear, with only 
+Base64 encoding which is easly decoded by someone able to see the traffic on the 
+connection. For that reason we only recommend Basic never be used on other than a 
+secure (https) conneciton.
+
+A more secure method of authentication uses Secure Protected Negotitation protocol, which enables
+negotiation of the security protcol from a selection of options, Currently the only option available with
+omi is the* NT Lan manager protocol, version 2 (NTLMV2)* which is an improvment over Basic auth in two ways. 
+
+- The password is hashed using an irreversable algorithm, so the password is more secure than Basic.
+
+- The client and server support encryption of your data over http connections, so SSL certificates are not required. 
+
+In order to use SPNEGO with NTLM, you will need to set up both the client and the server 
+to provide the credentials. 
+
+Note, NTLM support on gss is not available on MacOS. Solaris, Hpux, Aix or older linux distros
+can be modified to support it, but that configuration is not currently supported.  
+
+In the instructions following, the architecture x86_64 is assumed for simplicity. If you are
+using other architectures (for example,  Power8) please make the appropriate substitutions.
+
+###Setting up the machine
+OMI uses the MIt implementation of the Generic Security Services API (libgssapi_krb5.so). This is the 
+version installed by default in virtually all systems using anything like a Unix/linux OS. The MIT GSS 
```

Comment:
PM got me on that. They want UNIX/Linux due to some sort of trademark thing, I think.